app/models/feedback.rb

```diff
@@ -4,7 +4,7 @@ class Feedback < ApplicationRecord
 
   validates :rating, presence: true
 
-  scope :published_on, (->(date) { where('date(created_at) = ?', date) })
+  scope :published_on, (->(date) { where(created_at: date.all_day) })
```

Comment:
I did not know of this, nicee 👌

Reply:
Yeah, it's basically a shorthand for `date.beginning_of_day..date.end_of_day` (which is a mouthful). New in Rails > 5.1